src/inputs/input.css

```diff
@@ -170,3 +170,7 @@ input.omrs-input-filled:focus:not(.omrs-input-danger),
 input.omrs-input-outlined:focus:not(.omrs-input-danger) {
   border-color: var(--omrs-color-interaction);
 }
+
+input.omrs-input-outlined:focus {
+  border-width: 0.125rem;
```

Comment:
This [https://github.com/openmrs/openmrs-esm-styleguide/pull/32](PR) already has this change. Also is it active filled or active outlined?

Reply:
LGTM

Reply:
This is showing my lack of git skills. I created this pr and then created another branch for mf-48. I guess I'll close this PR and we can just merge the other one.